src/scripts/lexmatch-sssom-compare.py

```diff
@@ -173,11 +173,14 @@ def extract_unmapped_matches(input: str, matches: TextIO, output_dir: str, summa
     ont_df_list = []
 
     for _, ont in enumerate(input):
+        # Map ontology filenames to prefixes
         ont2 = ont.upper()
         if ont == "omim":
             ont2 = "|".join((["OMIM", "OMIMPS"]))
         elif ont == "ordo":
             ont2 = "|".join((["ORDO", "Orphanet"]))
+        elif ont == "icd11foundation":
```

Comment:
## ICD11 entry in `lexmatch-sssom-compare.py`
- [ ] @hrshdhgd @matentzn Review suggestions / etc
This was location 2 of 2 I updated to get ICD11 working in lexmatch. 

Nico mentioned that I should have >1k entries in lexmatch for this to be a success. I haven't ran `build-mondo-ingest` yet after this fix, but just from running the script, I got 7k files. I think Nico mostly means these files, both of which do have >1k entries each:
- `unmapped_icd11foundation_lex.tsv`
- `unmapped_icd11foundation_lex_exact.tsv`

I added documentation:
> ### 2.5. `lexmatch-sssom-compare.py`
> There is a section of branching logic with a comment "Map ontology filenames to prefixes". Add an entry there.

### Suggestions
I think probably branching logic inside of `lexmatch-sssom-compare.py` is not the best location for this kind of configuration. Thinking maybe moving to:
a. Some existing configuration file
  - a.a. `metadata/ONTOLOGY.yml`: My initial thought is add a `component_filename` field to `metadata/ONTOLOGY.yml`. And then we would have lexmatch simply check that, and then pull the `base_prefix_map`. Then we would pull `subject_prefixes` from there, although to do that, we'd have to go with [my suggestion](https://github.com/monarch-initiative/mondo-ingest/pull/434/files#r1531114745) of moving `subject_prefixes` from `mondo.config.sssom.yml`.
b. a new `lexmatch_config` (`.py`, `.json`, or `.yml`)

### FYIs
@hrshdhgd Was thinking about doing this in a separate PR but it is minor / we don't have to keep it. I also changed `combined_df = pd.concat(ont_df_list)` to `combined_df = pd.concat(ont_df_list) if ont_df_list else pd.DataFrame()`. This is because I got an error because I was running the script directly with just `icd11.foundation` as the only component, and since it didn't detect it, it got here and there was nothing to concat.

Reply:
New issue please

Reply:
Done!:
- #484

Reply:
@hrshdhgd Some minor comments I tagged you on here if you can take a look.

Reply:
got it! Thanks..